src/server/services/location.ts

```diff
@@ -51,25 +53,34 @@ export async function createPlaceIndex(): Promise<boolean> {
     await location.createPlaceIndex(INDEX_PARAMS).promise();
     return true;
   } catch (error) {
-    logger.error(`createPlaceIndex error: ${error.message}`);
+    const typedError = error as Error;
```

Comment:
I'm not sure what typescript is expecting.. aren't they all errors when caught? 🙄. anyway, thanks!

Reply:
🤷